src/Symfony/Component/JsonPath/Nothing.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\JsonPath;
+
+enum Nothing
```

Comment:
This enum should be `@internal`?

Reply:
No, it is needed by custom functions: people should be able to return this special Nothing value to be correctly handled by the component when necessary (PR to be merged, but we added it in prevision)